src/modality-specific-files/physiological-and-other-continuous-recordings.md

```diff
@@ -186,11 +300,342 @@ A guide for using macros can be found at
+-->
+{{ MACROS___make_columns_table("eyetrack.EyeTracking") }}
+
+Eye-tracking files `<matches>_recording-<label>_physio.tsv.gz` MAY be annotated
+with a corresponding `<matches>_recording-<label>_physioevents.tsv.gz` file.
+The `<matches>_recording-<label>_physioevents.tsv.gz` file MAY be employed to
+record discontinuous model parameters generated by the eye-tracker, for example,
+those derived from the saccade and blinks model some eye-trackers produce.
+
+**Example metadata sidecar**.
+Given the above example file structures, a corresponding
+`sub-01_task-visualSearch_recording-eye1_physio.json` sidecar
+could read:
+
+```JSON
+{
+    "DeviceSerialNumber": "17535483",
+    "Columns": ["x_coordinate", "y_coordinate", "timestamp", "blink"],
+    "EnvironmentCoordinates": "top-left",
+    "Manufacturer": "SR-Research",
+    "ManufacturersModelName": "EYELINK II CL v4.56 Aug 18 2010",
+    "RecordedEye": "right",
+    "SampleCoordinateSystem": "gaze-on-screen",
+    "SampleCoordinateUnits": "pixel",
+    "SamplingFrequency": 1000,
+    "SoftwareVersion": "SREB1.10.1630 WIN32 LID:F2AE011 Mod:2017.04.21 15:19 CEST",
+    "ScreenAOIDefinition": [
+        "square",
+        [100, 150, 300, 350]
+    ],
+    "blink": {
+      "Description": "Continuous recording of eyeblinks calculated by the eyetracking device (one indicates the eye is closed)."
+    }
+}
+```
+
+Content of `sub-01_task-VisualSearch_events.json`:
+
+```JSON
+{
+   "TaskName": "Visual Search",
+   "InstitutionName": "Stanford University",
+   "InstitutionAddress": "450 Serra Mall, Stanford, CA 94305-2004, USA",
+   "StimulusPresentation": {
+       "ScreenDistance": 60,
+       "ScreenRefreshRate": 60,
+       "ScreenResolution": [1024, 768],
+       "ScreenSize": [0.386, 0.29]
+   }
+}
+```
+
+**Example eye-tracking recording**.
+Given the above `sub-01_task-visualSearch_recording-eye1_physio.json` metadata
+specification, the decompressed content of the
+`sub-01_task-visualSearch_recording-eye1_physio.tsv.gz` can be:
+
+```TSV
+503    265    8506498    n/a
+510    235    8506499    n/a
+601    238    8506500    n/a
+550    243    8506501    n/a
+534    241    8506502    1
+520    237    8506503    1
+522    248    8506504    1
+520    250    8506505    n/a
+```
+
+Example `sub-01_task-visualSearch_recording-eye1_physioevents.tsv.gz` corresponding
+to the above eye-tracking recording, after decompressing:
+
+```TSV
+8478059 "NO Reply is disabled for function eyelink_cal_result"
+8499586 "RECCFG CR 1000 2 0 R"
+8499586 "ELCLCFG TOWER"
+8506500 "First task trigger"
+```
+
+where the first three rows are logged by the eye-tracker and the last row shows
+a *message* asynchronously received by the eye-tracker.
+The corresponding `sub-01_task-visualSearch_recording-eye1_physioevents.json` sidecar
+would read:
+
+```JSON
+{
+    "Columns": ["timestamp", "message"],
+    "Description": "Messages logged by the measurement device",
+    "ForeignIndexColumn": "timestamp"
+}
+```
```

Comment:
Note to self: add example datasets from the original standalone section. Apologies, I missed that

Reply:
Done